docs/src/reference/workflow-creation.md

```diff
@@ -0,0 +1,89 @@
+# Workflow Instance Creation
+
+Depending on the workflow definition, an instance of it can be created in various ways.
+
+## Explicit creation of a workflow instance
+
+A workflow instance can be created by sending a command specifying the process id or the unique key of the workflow.
+There are two commands to create a workflow instance.
+
+### Asynchronous execution
+
+ A workflow that has a [none start event](/bpmn-workflows/none-events/none-events.html#none-start-events) can be started explicitly using the command [CreateWorkflowInstance](/reference/grpc.html#createworkflowinstance-rpc).
+ When the broker receives this commands, it creates a new workflow instance and immediately respond with the workflow instance id.
+ The execution of the workflow happens after the response is send.
+
+ <details>
+   <summary>Code example</summary>
+   <p>Create a workflow instance:
+
+ ```java
+final WorkflowInstanceEvent workflowInstanceEvent =
+          zeebeClient              
+              .newCreateInstanceCommand()
+              .bpmnProcessId(bpmnProcessId)
+              .latestVersion()
+              .send()
+              .join();
+System.out.println(
+        "Workflow instance created with key: " + workflowInstanceEvent.getWorkflowInstanceKey());
+```
+
+   </p>
+ </details>
+
+### Create and await results
+
+Typically, workflow creation and execution are decoupled.
+However, there are use-cases that need to collect the results of a workflow when it's execution is completed.
+The [CreateWorkflowInstanceWithResult](/reference/grpc.html#createworkflowinstancewithresult-rpc) command allows you to “synchronously” execute workflows and receive the results via a set of variables.
```

Comment:
An image would help to understand it easier/faster.

Reply:
We should clarify a bit more that "synchronously" means here to not confuse the users.